build.gradle

```diff
@@ -0,0 +1,109 @@
+plugins {
+    id 'java'
+    id 'com.github.johnrengelman.shadow' version '1.2.3'
+}
+
+group 'com.scality.clueso'
+version '1.0-SNAPSHOT'
+
+apply plugin: 'java'
+apply plugin: 'scala'
+apply plugin: 'idea'
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+configurations {
+    provided
+    compile.extendsFrom provided
+}
+
+sourceSets {
+    main {
+        compileClasspath += configurations.provided
+
+        resources {
+            filter.exclude("application.conf")
+        }
+    }
+}
+
+repositories {
+    jcenter()
+    mavenLocal()
+    mavenCentral()
+    maven {
+        url "https://oss.sonatype.org/content/repositories/snapshots"
+    }
+}
+
+dependencies {
+    compile "org.scala-lang:scala-library:"+scalaVersion
+    compile "org.scala-lang:scala-reflect:"+scalaVersion
+    compile "org.scala-lang:scala-compiler:"+scalaVersion
+
+    compile 'org.apache.spark:spark-sql_2.11:2.1.0'
+    compile 'org.apache.spark:spark-streaming-kafka-0-10_2.11:2.1.0'
+    compile 'org.apache.spark:spark-sql-kafka-0-10_2.11:2.1.0'
```

Comment:
We need both: https://mvnrepository.com/artifact/org.apache.spark/spark-sql-kafka-0-10_2.11/2.1.0 
Spark SQL is expected to be provided in the same project, so the dependency of spark-sql-kafka doesn't 'pull' the spark-sql dependency.